### PR TITLE
fix: add autoplay for youtube and vimeo

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/6788
server PR : https://github.com/fossasia/open-event-server/pull/7775

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/forms/events/view/videoroom-form.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ export default class VideoroomForm extends Component.extend(FormMixin) {
         return;
       }
     }
-
+    this.data.stream.set('extra', { 'autoplay': false });
```
Comment (iamareebjamal) [inline app/components/forms/events/view/videoroom-form.js:146]: Bad

**Comment by mariobehling** [comment]
Please add screenshots.

**Comment by maze-runnar** [comment]
autoplay is working - 
Screenshot from 2021-02-24 22-20-15

Screenshot from 2021-02-24 22-20-51

**Comment by maze-runnar** [comment]
https://user-images.githubusercontent.com/56407566/109036943-29c5f580-76f0-11eb-9ba6-48e247be966a.mp4

**Comment by mariobehling** [comment]
In my test event the stream does not start automatically yet.

**Comment by maze-runnar** [comment]
> In my test event with Vimeo the stream does not start automatically yet.

Server PR also need to be merge for this to work. And is is not merged yet.

**Comment by iamareebjamal** [comment]
@maze-runnar Several unnecessary changes were there and several necessary changes weren't

**Pull request merged**